skills/RiskIdentification.md

```diff
@@ -0,0 +1,29 @@
+# Risk Identification
+* [Define the software-system or part of a system, which is to have risk based testing analysis (e.g. is it full, partial system, a component, an element, etc.)](DefineTheSoftwareSystemForRiskBasedTestingAnalysis.md)
+* [Identify ground rules and assumptions](IdentifyGroundRulesAndAssumptions.md)
+* [Identify risk by name and with a description](IdentifyRiskByNameAndWithADescription.md)
+
+### Our Working Definition:
+
+Risk Identification:\
+The process of examining the program areas and each critical technical process to identify and document the associated risk.
+
+### A context where I used this skill:
+
+I used this during the test impact I created for a test assignment. I try to identify risks by talking to several people. These can be the Business Analist, End User, Developers, Solution Designers.
+I write down the risks in a so called Risk List. This list is input for my [Risk Analyses](RiskAnalyses.md)
+
+### How I'd recommend someone learn this skill:
+
+Risk Identification comes to common sense. You talk to people who can provide you information about risk of a feature, application or solution. Collect the information and work further with that.
```

Comment:
I don't think we should trivialize this by suggesting it's common sense.  For some, it's not so common.  What kinds of things should you talk to people about?  How do we broach that topic?  What are good questions?